### ISSUE TITLE
background-size gets dropped if background has !important

### ISSUE THREAD

**Issue by ngyikp** [jakubpawlowicz/clean-css#433]
Node version: v0.10.35
cleancss version: 3.0.4

Test script:
```js
var CleanCSS = require('clean-css');

var test = function(source) {
	console.log(new CleanCSS().minify(source).styles);
};

// correct:
test('p {background:url("a.jpg"); background-color:#fff; background-size:10px 10px;}');
test('p {background:url("a.jpg"); background-color:#fff; background-size:10px 10px !important;}');
test('p {background:url("a.jpg") !important; background-color:#fff !important; background-size:10px 10px;}');
test('p {background-image:url("a.jpg") !important; background-color:#fff !important; background-size:10px 10px !important;}');

// fail:
test('p {background:url("a.jpg") !important; background-color:#fff !important; background-size:10px 10px !important;}');
```
Output:
```css
p{background:url(a.jpg) #fff;background-size:10px 10px}
p{background:url(a.jpg) #fff;background-size:10px 10px!important}
p{background:url(a.jpg) #fff!important}
p{background-image:url(a.jpg)!important;background-color:#fff!important;background-size:10px 10px!important}
p{background:url(a.jpg) #fff!important}
```
The last line should be
```css
p{background:url(a.jpg) #fff!important;background-size:10px 10px!important}
```

**Comment by jakubpawlowicz** [comment]
It is wrong indeed!

**Comment by jakubpawlowicz** [comment]
It's fixed in 3.0.6.
